imesec-full-stack.yml

```diff
@@ -50,19 +50,19 @@ services:
         constraints: [node.role == manager]
 
   nginx-website:
-    image: richarvey/nginx-php-fpm:latest
+    image: imesec/website
     deploy:
       replicas: 2
       restart_policy:
         condition: any
         delay: "0"
     environment:
+      VIRTUAL_PORT: 5000
```

Comment:
This is a FLUP (follow-up) to this [postmortem](https://wiki.imesec.ime.usp.br/books/core/page/postmortem-13042020-nginx-proxy-novo-site-do-imesec/edit)